src/components/molecules/AdminDateTime/AdminDateTime.tsx

```diff
@@ -33,9 +35,11 @@ export const AdminDateTime: React.FC<DateFieldProps> = ({
   errors,
   dateProps,
   timeProps,
+  dateFieldName,
+  timeFieldName,
 }) => {
-  const nameDate = `${name}Date`;
-  const nameTime = `${name}Time`;
+  const nameDate = dateFieldName || `${name}Date`;
+  const nameTime = timeFieldName || `${name}Time`;
```

Comment:
This makes it difficult to standardize across different forms. 

```suggestion
  const nameDate = `${name}Date`;
  const nameTime = `${name}Time`;
```

More unified approach would be for the form itself to take on the convention of `nameDate` and `nameTime` 
or just not use this component for the time being, hence the [NOTE] part, might as well have used [OOS]

> [NOTE]
This is just legacy way of "doing business". The next PR can re-use some of SpaceTimingForm components

https://github.com/sparkletown/sparkle/pull/2599#discussion_r757031336